src/edu/csus/ecs/pc2/core/model/IFileImpl.java

```diff
@@ -2,8 +2,6 @@
 
 import java.util.Base64;
 
-import edu.csus.ecs.pc2.core.model.IFile;
```

Comment:
the changes to this file seem unrelated to this ticket.

Reply:
The relationship of the changes in this file to the ticket is simply that the unchanged version of this file generates Errors in Eclipse -- and I didn't want to submit a PR with code that has Errors flagged in it.  I suspect the underlying issue might be that I'm developing in a different Eclipse environment -- my laptop, in Sharm -- from what I normally use.  However, I tried but failed to figure out what is different, and given the urgency for ACPC of getting this ticket resolved I had two choices -- check in code with Errors, or fix the item causing the Error.  (Yes, I could have filed a separate ticket just for this -- but I didn't have time to do that and get the code working and tested.